packages/fiori/src/IllustratedMessage.js

```diff
@@ -75,9 +75,12 @@ const metadata = {
 		 * <br><br>
 		 * <b>Note:</b> By default BeforeSearch illustration is loaded. When using illustration type
 		 * it have to be loaded separately (<code>import @ui5/webcomponents-fiori/dist/illustrations/BeforeSearch.js";</code>).
+		 *
+		 * When using an illustration type, other than the default, it should be loaded in addition:
+		 * <br>
+		 * <code>import @ui5/webcomponents-fiori/dist/illustrations/BeforeSearch.js";</code>
```

Comment:
BeforeSearch.js -> NoData.js (or some other illustration, as BeforeSearch is always loaded )